### DISCUSSION TITLE
how to add callout title to toc

### DISCUSSION THREAD

**Discussion by spiritoftime** [fuma-nama/fumadocs#653] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/da71e873-a9b6-4e43-be95-2df12351d4f6)

Hello! I have a callout and i am looking to add the title as part of the table of content
```
<Callout title="Question 1">
  Ali and Billy have money in the ratio of 5 : 4. After Ali gave Billy $20,they
  have an equal amount of money. How much money does Billy have in the end?
</Callout>
```
it seems that i need to add ## like ## What is Next? (not too sure why thats the case, a link to the docs on this would be appreciated, having skill issues in finding it) in order to make it appear in the table of content. h2/other header tags dont work

 however im having trouble figuring out how to do so. Would appreciate your kind help!

my markdown file: https://github.com/spiritoftime/psle-math/blob/main/open-math-prep/content/questions/constantTotal/question1.mdx

**Comment by fuma-nama** [comment, marked as answer]
The title field of callout isn't a heading searchable by a Remark Plugin, you have to explicitly specify a heading in the MDX/Markdown file, as the children of JSX elements or in the document body.

**Reply by spiritoftime** [reply]
ah okay, just to clarify when you say specify it as children in jsx you mean this heading right?
https://fumadocs.vercel.app/docs/ui/mdx/heading#usage

if yes i guess i will just copy the callout styling and make the callout component with it
thanks!

**Reply by fuma-nama** [reply]
```mdx
## Heading
```
